keras/layers/core.py

```diff
@@ -17,6 +17,19 @@
 import sys
 
 
+class Pass:
```

Comment:
Hi, its a good practice to inherit from object always:
http://stackoverflow.com/questions/4015417/python-class-inherits-object